README.md

```diff
@@ -296,55 +296,48 @@ networks: {
   },
 ```
 ## Upgrade contract
-1. Make sure that your deployments/[network] folder is empty/deleted
-2. Set following variables in .env.[network] file
-``` bash
-NO_PROXY = true           # Deploys only implementation of contract (without proxy contract)
-NO_DEPENDENCIES = true    # Doesn't deploy any dependencies of specified deploy script
-
-# optional based on which contract will be updated
-MANAGER_ADDRESS = 0xaddress           # for Account contract
-STAKED_CELO_ADDRESS = 0xaddress       # for RebasedStakedCelo contract
-ACCOUNT_ADDRESS = 0xaddress           # for RebasedStakedCelo contract
-MULTISIG_ADDRESS = 0xaddress          # for RebasedStakedCelo contract
-```
-3. Run command
+1. Make your changes to smart contract (eg Manager.sol)
+2. Run command
 ``` bash
 > yarn hardhat stakedCelo:deploy  --network [network] --show-stack-traces --tags [tag of deploy script] --from "[deployed address]" --use-private-key
 ``` 
 Example
 ``` bash
-> yarn hardhat stakedCelo:deploy  --network alfajores --show-stack-traces --tags Account --from "0x7E71FB21D0B30F5669f8F387D4A1114294F8E418" --use-private-key
+> yarn hardhat stakedCelo:deploy  --network alfajores --show-stack-traces --tags core --from "0x7E71FB21D0B30F5669f8F387D4A1114294F8E418" --use-private-key
+```
+Since contracts are being owned by MultiSig deployment will in the end FAIL since our deployer doesn't have rights to upgrade proxy. 
```

Comment:
```suggestion
Since contracts are owned by MultiSig, the deployment will in the end FAIL since our deployer doesn't have rights to upgrade proxy. 
```

Reply:
A few questions:
* Are there additional steps that need to be taken after the failure?
* Could we catch expected failures when we know we're upgrading and exit cleanly? 
* Does the failure short circuit anything else that we wanted to happen? E.g. if there are two contracts that need to be upgraded, will the deploy script halt after the first failed upgrade, or will it continue to the next deploy script (again failing as expected)?

For points 2. and 3., maybe something like [this](https://github.com/wighawag/hardhat-deploy/issues/70#issuecomment-781557350) would work?

Reply:
Unfortunately catchUnknownSigner doesnt seem to work, but I introduced our method catchNotOwnerForProxy. This allows us to skip all proxy upgrade related issues. Because of that even multiple changed contracts will be deployed in one go.